ethers-solc/src/compile/mod.rs

```diff
@@ -70,7 +70,7 @@ pub(crate) fn take_solc_installer_lock() -> std::sync::MutexGuard<'static, ()> {
 /// A list of upstream Solc releases, used to check which version
 /// we should download.
 /// The boolean value marks whether there was an error accessing the release list
-#[cfg(all(feature = "svm-solc"))]
+#[cfg(all(feature = "svm-solc", not(target_arch = "wasm32")))]
```

Comment:
Does anything in ethers-solc work inside of WASM, given it's calling the binary? Should we just make the crate not compile with ethers-rs if wasm is not configured?